### ISSUE TITLE
got error while opening aarch64 elf

### ISSUE THREAD

**Issue by HexRabbit** [igogo-x86/HexRaysPyTools#50]
I'm using the latest commit f3d46b7 on IDA 7.0 opening an aarch64 file,
got this error pop-up
```
notify_when(): Error occured while notifying object.
Traceback (most recent call last):
  File "D:/IDA 7.0/plugins\HexRaysPyTools\core\cache.py", line 70, in initialize_cache
    _init_demangled_names()
  File "D:/IDA 7.0/plugins\HexRaysPyTools\core\cache.py", line 56, in _init_demangled_names
    short_name = common.demangled_name_to_c_str(short_name)
  File "D:/IDA 7.0/plugins\HexRaysPyTools\core\common.py", line 100, in demangled_name_to_c_str
    "implemented ({}). You can do it by yourself or create an issue".format(name))
AssertionError: Replacement of demangled string by c-string for keyword `operatorXXX` is not yetimplemented (boost::operators_impl::operators_detail::empty_base<boost::posix_time::time_duration>::empty_base(void)). You can do it by yourself or create an issue
```
but x86_64 works smoothly

**Comment by Frago9876543210** [comment]
I have the same problem with IDA 7.2 on lastest commit (2826a7b0721d908f854296d5df1c1116cb8ef52c)
```
notify_when(): Error occurred while notifying object.
Traceback (most recent call last):
  File "C:/Program Files/IDA 7.2/plugins\HexRaysPyTools\core\cache.py", line 70, in initialize_cache
    _init_demangled_names()
  File "C:/Program Files/IDA 7.2/plugins\HexRaysPyTools\core\cache.py", line 56, in _init_demangled_names
    short_name = common.demangled_name_to_c_str(short_name)
  File "C:/Program Files/IDA 7.2/plugins\HexRaysPyTools\core\common.py", line 105, in demangled_name_to_c_str
    "implemented ({}). You can do it by yourself or create an issue".format(name))
AssertionError: Replacement of demangled string by c-string for keyword `operatorXXX` is not yetimplemented (std::regex_constants::operator~(std::regex_constants::match_flag_type)). You can do it by yourself or create an issue
```

**Comment by HexRabbit** [comment]
@Frago9876543210 
It's the different problem. For my case, the original code didn't expect string "operators" inside the class name ("operater" + "s"), and since it's not a valid operator type, the fix is to filter it, so the bug here has been solved by ac54ae3. And in your case, it's caused by a not handled operator type "operator~".
